templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,71 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Manufacturer List
+    </h1>
+  <h1>Manufacturer List
+    <a style="float: right" href="{% url "taxi:manufacturer-create" %}">+</a>
+  </h1>
```

Comment:
There are duplicate `<h1>` tags for 'Manufacturer List'. Consider removing one to avoid redundancy.